library/EasyPipeline/src/Easy.hs

```diff
@@ -188,15 +209,15 @@ desugarMinimalPrelude = desugarLibrary def
 
 -- | @contextifyGen@ is the generator function for the various contexitfy passes
 contextifyGen ::
-  (NonEmpty (NameSymb.T, [Sexp.T]) -> IO b) -> ByteString -> Options -> IO b
+  (NonEmpty (NameSymb.T, [Sexp.T]) -> IO b) -> ByteString -> Options pt pv -> IO b
```

Comment:
Personally, I would write `primTy` and `primVal` here, to be in line with the rest of the code.